chainsaw-demo/src/usecases.rs

```diff
@@ -0,0 +1,19 @@
+//
+
+//!
+//!
+//!
+
+#[cfg_attr(test, mockall::automock)]
+pub trait HelloUseCase: std::fmt::Debug + Send + Sync {
```

Comment:
Is there any potential for `Send + Sync` hurting us at any point?

Reply:
I was never entirely sure what implementing `Send + Sync` required in practice as one of them was unsafe. As the implementation is an empty struct, I assume it's fine?